src/epics/sync.js

```diff
@@ -33,9 +39,13 @@ export const syncStartEpic = (action$: ActionsObservable<any>) =>
     .ofType(SYNC_START)
     .filter(action => action.status === 'Start')
     .flatMap(action =>
-      Sync.start(action.meta).map(response => {
-        return deviceSyncUpdate(action.meta, response);
-      })
+      Sync.start(action.meta)
+        .flatMap(response => {
+          return Observable.of(deviceSyncUpdate(action.meta, response))
```

Comment:
this needed `flatMap` and `Observable.of`, modeling some of the other epics code, although still unclear to me why.